.circleci/config.yml

```diff
@@ -3,110 +3,232 @@ version: 2.0
 defaults: &defaults
   working_directory: /go/src/github.com/sylabs/singularity-cri
   docker:
-    - image: sylabsio/golang:1.11-stretch
+    - image: golang:1.11-stretch
 
 jobs:
   get_source:
     <<: *defaults
     steps:
       - checkout
-      - restore_cache:
-          key: dep-cache-{{ checksum "Gopkg.lock" }}
-          paths:
-            - vendor
       - run:
           name: Install dep tool
-          command: |-
+          working_directory: /
+          command: |
```

Comment:
Nitpick: you probably want to keep the block chomp here (and elsewhere), otherwise the CircleCI output will look a little odd because there will be a newline after the last command.

Reply:
You mean keep `|-` instead of `|`? I just followed circleCI examples on my own

Reply:
I see no difference between output with and without it

Reply:
OK, it's possible they are chomping in CircleCI itself, it used to leave the extra newline, which messed with the output. Not a big deal either way.